lmorpho/morphology.cpp

```diff
@@ -0,0 +1,56 @@
+#include <cmath>
+#include <vector>
+
+#include <math.hpp>
+
+#include "morphology.hpp"
+
+using nest::mc::math::lerp;
+
+static section_point lerp(const section_point& a, const section_point& b, double u) {
```

Comment:
'Lerp' stands for linear interpolation; there is a bit more detail in `src/math.hpp`, but it's a fairly standard term.